integration-tests/clipper_admin_tests.py

```diff
@@ -674,6 +674,28 @@ def test_deployed_model_queried_successfully(self):
         self.assertNotEqual(parsed_response["output"], self.default_output)
         self.assertFalse(parsed_response["default"])
 
+    def test_deployed_model_queried_with_training_slash_successfully(self):
```

Comment:
Why don't we add a check in `test_test_predict_function` instead of creating a new test?

Reply:
Good. I was thinking a little bit about whether to add a new test.